lib/src/tasks/coverage/api.dart

```diff
@@ -214,13 +214,45 @@ class CoverageTask extends Task {
 
   Future _collect() async {
     List<File> collections = [];
+    PubServeTask pubServeTask;
+    PubServeInfo serveInfo;
+
+    if (pubServe && pubServeTask == null) {
```

Comment:
Oops, forgot to remove this check. At first it meant to start pub serve on demand if current test is browser test and no pub serve was running. I'll fix this and other issues tommorow. Thanks for feedback.